packages/venia-ui/lib/components/RichContent/PageBuilder/resolveLinkProps.js

```diff
@@ -0,0 +1,26 @@
+export default (link, linkType) => {
+    let isExternalUrl;
+    const linkProps = {};
+
+    try {
+        const baseUrl = document
+            .querySelector('link[rel="preconnect"]')
+            .getAttribute('href'); // TODO - some better way to get this?
```

Comment:
You may be able to get it from `.env`. Take a look at one of our other URL generating utilities.